federation/cluster/federation-up.sh

```diff
@@ -82,7 +82,8 @@ function init() {
       "${FEDERATION_NAME}" \
       --host-cluster-context="${HOST_CLUSTER_CONTEXT}" \
       --dns-zone-name="${DNS_ZONE_NAME}" \
-      --image="${kube_registry}/hyperkube-amd64:${kube_version}"
+      --image="${kube_registry}/hyperkube-amd64:${kube_version}" \
+      --apiserver-arg-overrides="--storage-backend=etcd2"
```

Comment:
Now that ``kubefed init`` is no longer specifying ``etcd2`` as the default, a user will get the underlying default of ``etcd3`` unless they specify the same override as appears here.  Why is it desirable that ``federation-up.sh`` not use ``etcd3`` as well?